src/components/fields/FieldInput.tsx

```diff
@@ -1,4 +1,4 @@
-import { FC, ReactNode } from "react";
+import type { FC, ReactNode } from "react";
```

Comment:
Interesting, do you know why biome suggests this to be imported?

Reply:
This is not an import, this is marking an import as a type import only, read more here: https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-8.html